### PR TITLE
update hardhat docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/708/files

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

https://github.com/moonbeam-foundation/moonbeam-docs/pull/719

### CODE REVIEW

File: builders/build/eth-api/verify-contracts/etherscan-plugins.md [hunk 1]
```diff
@@ -51,21 +49,19 @@ Hardhat插件可以无缝集成至您的[Hardhat](https://hardhat.org/){target=_
...
 
-```
-npm install --save-dev @nomiclabs/hardhat-etherscan
+```bash
+npm install --save-dev @nomicfoundation/hardhat-verify
 ```
 
 !!! 注意事项
-    `@nomiclabs/hardhat-etherscan`的3.0.1版本中已添加对基于Moonbeam网络的支持。您可以通过查看`devDependencies`部分的`package.json`来确认所使用的版本并更新至3.0.1或以上的版本（若需要）。
+    `@nomiclabs/hardhat-verify`的3.0.1版本中已添加对基于Moonbeam网络的支持。您可以通过查看`devDependencies`部分的`package.json`来确认所使用的版本并更新至3.0.1或以上的版本（若需要）。
```
Comment (themacexpert) [inline builders/build/eth-api/verify-contracts/etherscan-plugins.md:57]: Should this also get changed to nomic foundation here or it's good as labs?

**Review by themacexpert — APPROVED** [review]
LGTM 👍

File: builders/build/eth-api/verify-contracts/etherscan-plugins.md (hunk 1 above)
Comment (eshaben) [reply builders/build/eth-api/verify-contracts/etherscan-plugins.md:57]: good catch! yes it should thanks 🙂

Comment (eshaben) [inline builders/build/eth-api/verify-contracts/etherscan-plugins.md:57]:
```suggestion
    `@nomicfoundation/hardhat-verify`的3.0.1版本中已添加对基于Moonbeam网络的支持。您可以通过查看`devDependencies`部分的`package.json`来确认所使用的版本并更新至3.0.1或以上的版本（若需要）。
```

**Pull request merged**